scss/angular/overrides/_datepicker.scss

```diff
@@ -0,0 +1,137 @@
+[uib-datepicker-popup] {
```

Comment:
the name of the file should be uib-datepicker, otherwise it could be *any* datepicker